pkg/chaosdaemon/iochaos_server.go

```diff
@@ -130,7 +132,7 @@ func (s *DaemonServer) ApplyIOChaos(ctx context.Context, in *pb.ApplyIOChaosRequ
 	}, nil
 }
 
-func (s *DaemonServer) killIOChaos(ctx context.Context, pid int64, startTime int64) error {
+func (s *DaemonServer) killIOChaos(ctx context.Context, log logr.Logger, pid int64, startTime int64) error {
```

Comment:
That's a little weird, we could access two loggers inside method `killIOChaos`.

I prefer to also call `getLoggerFromContext` inside of `killIOChaos`

Reply:
done